fiftyone/plugins/core.py

```
@@ -67,12 +70,17 @@ def list_plugins(enabled=True, builtin=False):
+    if shadowed == "all":
+        shadowed = None
+
     plugins = []
-    for p in _list_plugins(enabled=enabled, builtin=builtin):
+    for p in _list_plugins(
+        enabled=enabled, builtin=builtin, shadowed=shadowed
+    ):
         try:
-            plugins.append(_load_plugin_definition(p))
+            plugins.append(_load_plugin_definition(plugin=p))
         except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)